contentful/export-processor/content-types/section.js

```diff
@@ -191,10 +187,10 @@ export class Section {
                 const unusedAnswers = currentQuestion.answers.filter(answer => !answersUsed.includes(answer.id));
 
                 // Find answers that lead to the next question in the sequence, prioritising unused answers and skipping any that jump ahead
-                if (unusedAnswers.some(answer => answer.nextQuestion?.sys.id === nextNeeded)) {
-                    currentAnswer = unusedAnswers.find(answer => answer.nextQuestion.sys.id === nextNeeded);
-                } else if (currentQuestion.answers.some(answer => answer.nextQuestion.sys.id === nextNeeded)) {
-                    currentAnswer = currentQuestion.answers.find(answer => answer.nextQuestion.sys.id === nextNeeded);
+                if (unusedAnswers.some(answer => answer.nextQuestion.id === nextNeeded)) {
```

Comment:
If it is null then we don't want to select it, because we need one that leads us to the one we just set (the last question in the section with unused answers).

Reply:
But won't

```
answer => answer.nextQuestion.id === nextNeeded
```

This error if nextQuestion is falsy then?

That said, it didn't error when I ran it - so I assume there's something that has already filtered out the answers with next questions at this point, but it's not clear/I'm missing it :D

Reply:
I think because it's in the unusedAnswers.some() so if all the unused answers are falsy it'll move on?